dspace-api/src/main/java/org/dspace/util/SimpleMapConverter.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.util;
+
+import org.apache.commons.lang3.StringUtils;
+import org.dspace.submit.lookup.MapConverterModifier;
+
+/**
+ *
+ * @author Mykhaylo Boychuk (mykhaylo.boychuk at 4science.it)
+ */
+public class SimpleMapConverter extends MapConverterModifier {
+
+    public String getValue(String key) {
```

Comment:
Please add JavaDocs to describe the logic in this method.  It also appears this may not have tests (according to Codecov).